_includes/release-notes/overview.md

```diff
@@ -8,7 +8,7 @@ TinyMCE 5.0 has an ARIA [WCAG 2.1 standards](https://www.w3.org/WAI/standards-gu
 Include this line of code in the <head> of the HTML page and link to the tinymce.min.js source file:
 
 ```js
-<script src="{{ site.cdnurl }}?apiKey=your_API_key"></script>
+<script src="{{ site.cdnurl }}"></script>
```

Comment:
@mhideo This one doesn't explain `no-api-key` that I can see.

Reply:
Added: http://docs-v5-feature-tcloud-2861-2.staging.tiny.cloud/docs/release-notes/release-notes50/#overview